compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2785,7 +2785,7 @@ fn check_invalid_crate_level_attr(tcx: TyCtxt<'_>, attrs: &[Attribute]) {
             .map(|id| tcx.hir_item(id))
             .find(|item| !item.span.is_dummy()) // Skip prelude `use`s
             .map(|item| errors::ItemFollowingInnerAttr {
-                span: item.ident.span,
+                span: item.kind.ident().unwrap().span,
```

Comment:
I wonder if this unwrap will panic on code like this:

```rs
#![automatically_derived]
impl Ty {} // <- `item` if I'm not mistaken
struct Ty;
```

Reply:
It does, good catch. I have adjusted it to fall back to `item.span`.